src/pages/concept-registration-page/form-term/form-term.component.tsx

```diff
@@ -12,6 +12,25 @@ interface FormProps {
 
 type FormValues = Pick<Concept, 'anbefaltTerm' | 'definisjon' | 'kildebeskrivelse' | 'merknad'>;
 
+const preProcessValues = origValues => {
+  console.log('*** origValues.kildebeskrivelse: ' + JSON.stringify(origValues, null, 2));
+
+  if (_.get(origValues, 'kildebeskrivelse') == null) {
+    return origValues;
+  }
+  const kildes = origValues.kildebeskrivelse.kilde;
+  const nonEmptyKildes = _.filter(kildes, kilde => !!(kilde.uri || kilde.tekst));
+  const values = _.cloneDeep(origValues);
+  values.kildebeskrivelse.kilde = nonEmptyKildes;
+  return values;
+};
```

Comment:
It is not so easy to understand the purpose of this function

Reply:
Added comments.